stages/org.osbuild.ovf.meta.json

```diff
@@ -18,6 +18,31 @@
           "description": "The vmdk image filename present in the root of the tree",
           "type": "string",
           "pattern": "[a-zA-Z0-9+_.-]+.vmdk"
+        },
+        "vmware": {
+          "description": "Settings for VMWare specific parts of the OVF file.",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "os_type": {
+              "type": "string",
+              "default": "other26xLinux64Guest"
+            }
+          }
+        },
+        "virtualbox": {
+          "description": "Settings for VirtualBox specific parts of the OVF file.",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "os_type": {
+              "type": "string",
+              "default": "OtherLinux_64"
+            },
+            "mac_address": {
+              "type": "string"
```

Comment:
Should we be more precise here using e.g. a regexp? Or are we lax in similar situations already?

Reply:
I was thinking about this as well, we *could* use a regexp, I'll add one.